src/App.jsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import './App.scss';
+import Sum from './components/Sum';
 
 const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove comment lines :) */}
+    <Sum a={2} b={5} />
+    <Sum a={-5} b={0} />
```

Comment:
no need to pass zero value since you're already handling it when providing default value for a prop in `Sum` component
```suggestion
    <Sum a={-5} />
```